cloudsim-plus-testbeds/src/main/java/org/cloudsimplus/migration/ExamplePowerVmSelectionPolicyMinimumUtilization.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package org.cloudsimplus.migration;
+
+import java.util.List;
+import org.cloudbus.cloudsim.hosts.power.PowerHost;
+import org.cloudbus.cloudsim.selectionpolicies.power.PowerVmSelectionPolicy;
+import org.cloudbus.cloudsim.vms.Vm;
+import org.cloudbus.cloudsim.vms.power.PowerVm;
+
+/**
+ *
+ * @author raysaoliveira
+ */
+public class ExamplePowerVmSelectionPolicyMinimumUtilization extends PowerVmSelectionPolicy {
```

Comment:
Again, you are duplicating an entire class from the core. This class must be deleted and the original class used without changes. In fact, it doesn't appear you added any code to this class.